x-pack/plugins/security_solution/server/endpoint/routes/resolver/tree/queries/stats.ts

```diff
@@ -94,6 +95,70 @@ export class StatsQuery {
     };
   }
 
+  private alertStatsQuery(
+    nodes: NodeID[],
+    index: string,
+    includeHits: boolean
+  ): { size: number; query: object; index: string; aggs: object; fields?: string[] } {
+    if (includeHits) {
+      return {
+        size: 5000,
```

Comment:
Can we just return a single object with `size: includeHits ? 5000 : 0` since everything else is duplicated?